modules/installing-extensions-with-code-server.adoc

```diff
@@ -0,0 +1,36 @@
+:_module-type: PROCEDURE
+
+[id="installing-extensions-with-code-server_{context}"]
+= Installing extensions with code-server
+
+With the code-server workbench image, you can customize your code-server environment by using extensions to add new languages, themes, and debuggers, and to connect to additional services. You can also enhance the efficiency of your data science work with extensions for syntax highlighting, auto-indentation, and bracket matching. 
```

Comment:
This is super minor consideration, but this paragraph is very poor elevator pitch for code-server. It's like selling people cars and promoting how they'll be able to equip them with jiggly dashboard toys of various colors (instead of saying that they'll be able to transport family and items).

Reply:
Actually, this is rightly praising extensibility because it's in the extensions section. Ok, please disregard.